Enterprise-CMCS/macpro-mdct-mcr
# Improve pdf accessibility

## Summary
Added more accessibility elements for PDF. 

### Description

- Make Excel workbook icons decorative by setting alt-text to an empty string
- Remove header and footer (URL/page number) on print
- Remove repetitious column titles when tables span multiple pages
- Concatenate dynamic fields so that navigating by header includes entire string

### Related ticket
[MDCT-2292]


### How to test

1. Go to print report page on any MCPAR report
2. Open console ( I used chrome)
3. Turn on Accessibility Tree View

Testing "Make Excel workbook icons decorative by setting alt-text to an empty string"
1. Click on excel icon on the print page, the accessibility tree view should show ignored

Testing "Concatenate dynamic fields so that navigating by header includes entire string"
1. Click on the title text of the form "Managed Care Program Annual Report (MCPAR) for Minnesota: [name]"
2. In accessibility tree view, when the heading is expanded, there should only be a single StaticText showing the title

Testing "Remove header and footer (URL/page number) on print"
1. Click [Download PDF], in the print viewer, there shouldn't be any more links in the header or footer

Testing "Remove repetitious column titles when tables span multiple pages"
1. Click [Download PDF], in the print viewer, table headers will not spill over to the next print page when the table is split between pages

### Important updates

N / A

### Author checklist
